ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm

```diff
@@ -447,3 +447,598 @@
+					if(T.density)
+						break
+					for(var/turf/Y in getline(T, get_ranged_target_turf(T, NORTH, smash_width)))
+						if (Y.density)
+							break
+						if (Y in area_of_effect)
+							continue
+						area_of_effect += Y
+					for(var/turf/U in getline(T, get_ranged_target_turf(T, SOUTH, smash_width)))
+						if (U.density)
+							break
+						if (U in area_of_effect)
+							continue
+						area_of_effect += U
+				source_turf = get_ranged_target_turf(second_line, WEST, smash_length)
+				smash_length += 2
+				smash_width ++
+		if(NORTH)
+			for(var/i = 0, i<2, i++)
+				middle_line = getline(source_turf, get_ranged_target_turf(source_turf, dir_to_target, smash_length))
+				for(var/turf/T in middle_line)
+					if(T.density)
+						break
+					for(var/turf/Y in getline(T, get_ranged_target_turf(T, EAST, smash_width)))
+						if (Y.density)
+							break
+						if (Y in area_of_effect)
+							continue
+						area_of_effect += Y
+					for(var/turf/U in getline(T, get_ranged_target_turf(T, WEST, smash_width)))
+						if (U.density)
+							break
+						if (U in area_of_effect)
+							continue
+						area_of_effect += U
+				source_turf = get_ranged_target_turf(second_line, NORTH, smash_length)
+				smash_length += 2
+				smash_width ++
+		if(SOUTH)
+			for(var/i = 0, i<2, i++)
+				middle_line = getline(source_turf, get_ranged_target_turf(source_turf, dir_to_target, smash_length))
+				for(var/turf/T in middle_line)
+					if(T.density)
+						break
+					for(var/turf/Y in getline(T, get_ranged_target_turf(T, EAST, smash_width)))
+						if (Y.density)
+							break
+						if (Y in area_of_effect)
+							continue
+						area_of_effect += Y
+					for(var/turf/U in getline(T, get_ranged_target_turf(T, WEST, smash_width)))
+						if (U.density)
+							break
+						if (U in area_of_effect)
+							continue
+						area_of_effect += U
+				source_turf = get_ranged_target_turf(second_line, SOUTH, smash_length)
+				smash_length += 2
+				smash_width ++
+
+		else
+			for(var/turf/T in view(1, src))
+				if (T.density)
+					break
+				if (T in area_of_effect)
+					continue
+				area_of_effect |= T
+
+	if (!LAZYLEN(area_of_effect))
+		return
+	for(var/turf/T in area_of_effect)
+		new /obj/effect/temp_visual/lovetown_whip(T)
+		HurtInTurf(T, list(), lovewhip_damage, RED_DAMAGE, check_faction = TRUE, hurt_mechs = TRUE)
+		for(var/mob/living/L in T)
+			var/atom/throw_target = get_edge_target_turf(L, get_dir(L, src))
+			L.throw_at(throw_target, 200, 4)
+
+	playsound(get_turf(src), 'sound/creatures/lc13/lovetown/abomination_lovewhip_hit.ogg', 75, 0, 3)
+	SLEEP_CHECK_DEATH(1 SECONDS)
+	icon_state = icon_living
+	can_act = TRUE
+	if(prob(20)) //small chance to go into a dash after
+		DashCounter()
+	else
+		DisableCounter(original_speed)
+	return
+
+/mob/living/simple_animal/hostile/lovetown/abomination/proc/Finisher(mob/living/carbon/human/TH) //return TRUE to prevent attacking, as attacking causes runtimes if the target is gibbed.
```

Comment:
Doesn't take into account the potential for the target being moved away in the ~2.5 second windup which could be weird.

Reply:
Made it so the execute cancels if you are dragged away.

Reply:
Unfathomably based design, honestly. Team work foiling enemies is always fun.